src/Symfony/Component/Mailer/Exception/UnsupportedSchemeException.php

```diff
@@ -56,6 +56,10 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\OhMySmtp\Transport\OhMySmtpTransportFactory::class,
             'package' => 'symfony/oh-my-smtp-mailer',
         ],
+        'infobip' => [
```

Comment:
Can you please sort this alphabetically? Thanks

Please move ohmysmtp, too

Thanks